src/VaultV2.sol

```diff
@@ -68,6 +68,8 @@ import {ISharesGate, IReceiveAssetsGate, ISendAssetsGate} from "./interfaces/IGa
 /// the same ids.
 /// @dev If allocations underestimate the actual assets, some assets might be lost because deallocating is impossible if
 /// the allocation is zero.
+/// @dev On allocation or deallocation, the underlying position of an adapter may become underestimated, for instance due to roundings or entry/exit fees.
+/// @dev This understimate is only resolved in the next transaction, and so it should be small to avoid excessive fees and withdrawals at unfair share prices.
```

Comment:
the main problem is the fact that allocators can loop allocate/deallocate, so it should stay small notably compared to the gas (the comment right now don't make me think about that)